Assignment_2/Assignment_2/Models/Person.h

```diff
@@ -0,0 +1,14 @@
+
+
+#import <Foundation/Foundation.h>
+#import "Address.h"
+
+
+
+@interface Person : NSObject
+@property NSString * firstName;
+@property NSString * lastName;
+@property Address *address;
+
+-(Person *)initWithValuesOfCity:(NSString *) city withState :(NSString *) state;
```

Comment:
use instanceType rather than (Person*) itself, create its init only without parameters, and 
give their initial values as empty strings